.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,31 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: windows-2019
+
+    strategy:
+      matrix:
+        node-version: [6.x, 10.x]
+
+    steps:
+    - name: Configure git line-breaks
+      run: git config --global core.autocrlf false
+    - name: Checkout repository
+      uses: actions/checkout@v1
+    - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
The formatting of this step is completely broken, though it seems to use the correct Node version